frontend/src/views/groups/tabs/modal/Toggle.tsx

```diff
@@ -0,0 +1,50 @@
+import { AiFillInfoCircle } from "react-icons/ai";
+
+interface ToggleProps {
+  description?: string;
+  label: string;
+  value: boolean;
+  changeHandler: () => void;
+}
+
+export const Toggle = ({
+  description,
+  label,
+  value,
+  changeHandler,
+}: ToggleProps) => (
+  <div className="flex items-center justify-between relative">
+    { description ?
+    <div className="group relative overflow-visible">
+      <div className="absolute top-5 z-10 items-center opacity-0 group-hover:opacity-100 transition-opacity border-gray-200 rounded-2xl bg-white border-2">
```

Comment:
Using opacity and group hover like this means that when you hover over the hidden content it will also show, which is not wanted functionality. By replacing `opacity-0 group-hover:opacity-100` with `hidden group-hover:block`, it would work as intended. Then it only shows when hovering over the label and when hovering over the content only when visible.